src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,26 @@
 package mate.academy.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
+    @Size(max = 255, message = "Text limit 255 symbols")
```

Comment:
This message is repeated, you can make it a constant

Reply:
It is not critical for messages, but it is good point =)